src/views/publish/dimension-revisit.ejs

```diff
@@ -17,7 +17,7 @@
         <% if (locals?.data) { %>
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-full with-overflow">
-                    <%- include("../partials/dimension-preview-table.ejs", { dimension: locals.dimension, headers: locals.headers, data: locals.data }) %>
+                    <%- include("../partials/view-table.ejs", { dimension: locals.dimension, headers: locals.headers, data: locals.data }) %>
```

Comment:
I don't think this path exists on the publish side? Is there a missing file from the commit, or maybe an overly greedy search-replace?